Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -75,8 +150,19 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```

Comment:
Reuse InsertAt

Reply:
Just made requested changes.

Reply:
You should name those changes. AND you should resolve the remarks :smi